test/integration_test.go

```diff
@@ -149,6 +150,47 @@ func TestLogWithAbsPath(t *testing.T) {
 	}
 }
 
+func TestLogWithRelPath(t *testing.T) {
+	t.Parallel()
+
+	cleanupTerraformFolder(t, testFixtureLogRelPaths)
+	tmpEnvPath := copyEnvironment(t, testFixtureLogRelPaths)
+	rootPath := util.JoinPath(tmpEnvPath, testFixtureLogRelPaths)
+
+	testCases := []struct {
+		workingDir string
+		assertFn   func(t *testing.T, stdout, stderr string)
+	}{
+		{
+			workingDir: "duplicate-dir-names/workspace/one/two/aaa", // dir `workspace` duplicated twice in path
+			assertFn: func(t *testing.T, _, stderr string) {
+				t.Helper()
+
+				assert.Contains(t, stderr, "Module ./bbb/ccc/workspace")
+				assert.Contains(t, stderr, "Module ./bbb/ccc/module-b")
+				assert.Contains(t, stderr, "Downloading Terraform configurations from .. into ./bbb/ccc/workspace/.terragrunt-cache")
+				assert.Contains(t, stderr, "[bbb/ccc/workspace]")
+				assert.Contains(t, stderr, "[bbb/ccc/module-b]")
+			},
+		},
+	}
+
+	for i, testCase := range testCases {
+		testCase := testCase
```

Comment:
@levkohimins you don't need to do this anymore, as of go 1.22:
https://tip.golang.org/doc/go1.22

Reply:
Understood. Fixed. Thanks.